@here/harp-examples/src/camera-animations_key-track.ts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
```

Comment:
Sorry frauke, advised us that this needs to be 2017-2020, see the message from Ignacio in the Core Team webex chat.